README.md

```diff
@@ -37,8 +42,27 @@ outputs:
   branch_id:
     description: 'Reset branch id'
     value: ${{ steps.reset-branch.outputs.branch_id }}
+  db_url:
+    description: 'DATABASE_URL of the branch after the reset'
+    value: ${{ steps.reset-branch.outputs.db_url }}
+  db_url_with_pooler:
+    description: 'DATABASE_URL with pooler of the branch after the reset'
+    value: ${{ steps.reset-branch.outputs.db_url_with_pooler }}
+  cs_role_name:
```

Comment:
I don't see where these values are added as outputs. I am not sure if it makes sense, too.

Reply:
I've added `host`, `host_with_pooler`, `password` - same as for create-branch-action.